src/main/scala/ru/org/codingteam/horta/protocol/jabber/MucMessageHandler.scala

```diff
@@ -164,5 +164,23 @@ class MucMessageHandler(locale: LocaleDefinition,
       core ! CoreParticipantLeft(Clock.now, roomJID, participantJID, reason, self)
     }
   }
+}
+
+object MucMessageHandler {
 
+  private val vowels = "(?i)[aeiouаоэиуыеёюя]".r // TODO: Use some phonetic definition instead of hardcode
+  private val dash = "-"
```

Comment:
Why didn't you call the variable `separator` or `delimiter`? I mean what's the point to call the variable by its value. I'd rather call it by its purpose.